validation-test/compiler_crashers_fixed/28723-unreachable-executed-at-swift-lib-sema-csdiag-cpp-4012.swift

```diff
@@ -6,6 +6,7 @@
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
 // rdar://56144412
+// XFAIL: asserts
```

Comment:
Does this mean that this case started crashing again and has to be moved to "compiler_crashers"? 😢